src/deferred.js

```diff
@@ -385,5 +394,12 @@ jQuery.extend( {
 	}
 } );
 
+function deferredTick() {
+	processQueueTimeout = undefined;
+	jQuery.each( processQueue.splice( 0 ), function( _, process ) {
```

Comment:
> Is the test I added in 4bc074b correct? One notify throwing should not prevent the second...?

That achieves the goal of getting all the notifies to run but it swallows an exception in the process.  Maybe we've reached a choice between the lesser of two evils, I dunno. If throwing on a notify doesn't prevent Promise/A+ compat we should allow it. Swallowing the error seems really undesirable though.